casepro/pods/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from casepro.pods.views import perform_pod_action, read_pod_data
+
+urlpatterns = (
+    url(r'read/(?P<pk>\d+)/$', read_pod_data, name='read_pod_data'),
```

Comment:
Might be better to reference pods by a configurable slug rather than a numerical index which is tied to the ordering of settings.POD, no? You guys are the ones using this so your call.